Formula/libtomcrypt.rb

```diff
@@ -0,0 +1,29 @@
+class Libtomcrypt < Formula
+  desc "Comprehensive, modular and portable cryptographic toolkit"
+  homepage "http://www.libtom.net"
+  url "https://github.com/libtom/libtomcrypt/archive/v1.18.0.tar.gz"
+  sha256 "bdba1499dab3bf3365fa75553f069eba7bea392e8f9e0381aa0e950a08bd85ba"
+
+  option "with-gmp", "enable gmp as MPI provider"
+  option "with-libtommath", "enable libtommath as MPI provider"
+
+  depends_on "gmp" => :optional
+  depends_on "libtommath" => :optional
+
+  def install
+    ENV.append "CFLAGS", "-DUSE_GMP -DGMP_DESC" if build.with?("gmp")
+    ENV.append "EXTRALIBS", "-lgmp" if build.with?("gmp")
+    ENV.append "CFLAGS", "-DUSE_LTM -DLTM_DESC" if build.with?("libtommath")
+    ENV.append "EXTRALIBS", "-ltommath" if build.with?("libtommath")
+    system "make", "install", "PREFIX=#{prefix}"
+    system "make", "test"
```

Comment:
Seems like it should go before `make install`?

Reply:
`make install PREFIX=...` only installs the library and headers, so it doesn't matter whether `make test` is run before or after.

Reply:
`make install` is usually the last step, but yes, it's unhelpful `make test` is named that way & doesn't actually test anything at that point, presumably.